core/src/main/java/io/questdb/griffin/engine/functions/str/PadFunctionFactory.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.functions.str;
+
+import io.questdb.cairo.CairoConfiguration;
+import io.questdb.cairo.sql.Function;
+import io.questdb.cairo.sql.Record;
+import io.questdb.griffin.FunctionFactory;
+import io.questdb.griffin.SqlExecutionContext;
+import io.questdb.griffin.engine.functions.StrFunction;
+import io.questdb.griffin.engine.functions.BinaryFunction;
+import io.questdb.std.IntList;
+import io.questdb.std.ObjList;
+import io.questdb.std.Chars;
+import io.questdb.std.Numbers;
+import io.questdb.std.str.StringSink;
+import org.jetbrains.annotations.Nullable;
+
+public class PadFunctionFactory implements FunctionFactory {
+    @Override
+    public String getSignature() {
+        return "pad(SI)";
```

Comment:
This function is absent in Postgres and I failed to find it in other well-known databases.

@pswu11 @javier could you tell me why do we have it in #2578?

Reply:
this is very likely my mistake. I suggested lpad and rpad and very probably I added pad without thinking about it. Sorry!

Reply:
Good to know. We should remove `pad()` from this PR then.

Reply:
Thanks for spotting this! 🙏🏼

Reply:
done